lua/entities/ent_fedora_point.lua

```diff
@@ -0,0 +1,34 @@
+AddCSLuaFile()
+
+ENT.Type = "anim"
+DEFINE_BASECLASS("base_gmodentity")
+
+function ENT:Initialize()
+	if CLIENT then
+		self.Entity:SetNoDraw(true)
+	else
+		self.Trail = util.SpriteTrail(self, 0, Color(255, 255, 255), false, 4, 4, 0.2, 1/8, "chev/rainbowdashtrail") --trail is parented to this entity
+	end
+end
+
+function ENT:Think()
+	if SERVER then return end

```

Comment:
I think you need to set the entity's position to the player's GetPos serverside, because the server doesn't network the existence of new entities that aren't in a client's PVS (ie that the server doesn't think the client can see)

Reply:
Ok, fixed. It should network to the player now since it's set to the player's position in the think hook

Reply:
nvm it shows up in first person again, i'll have to fix that

Reply:
Ok, I made it so the entity networks to clients regardless of if it's in the PVS or not, it's a cheap fix but it should work